### PR TITLE
Update execution-environment to support current requirements.txt

### LINKED ISSUE #75: opcap-ansible-ee-build.yml failing due to requirements.txt pinned versions
The last GitHub workflows [opcap-ansible-ee-build.yml](https://github.com/opdev/opcap-ansible/blob/main/.github/workflows/opcap-ansible-ee-build.yml) job that ran 3 weeks ago failed due to inability to install pinned versions of [requirements.txt](https://github.com/opdev/opcap-ansible/blob/main/requirements.txt). Only older versions of these dependencies are available on the builder base image. 

Upon investigating the issue, discovered that upstream ansible-builder's base image relies on [ansible-runner](https://quay.io/ansible/ansible-runner) image which hasn't been updated in over a year. This image still using python 3.8 and will only allow older `requirements.txt` versions. 

It would be nice to not need to maintain two `requirements.txt` files or modify this base image with later version of python.

Thinking we should switch over to downstream ansible-builder which uses ee-minimal-rhel8 base image. It is well maintained and should allow us to install all dependencies from current `requirements.txt`.

### PR DESCRIPTION
**Pull request by madorn**
Resolves #75.

Updated associated GH workflow.

### CODE REVIEW

**Comment by bcrochet** [comment]
In order for this to be ok, we can't publish to quay.io. We can only publish to a private automation hub.

**Comment by madorn** [comment]
Per team discussion, we will push this image to our internal PAH repo. But also want to see if we can find out if another upstream image resides somewhere other than the quay ansible-runner repo. 

I have reached out to the ansible community to find out.

**Comment by madorn** [comment]
https://forum.ansible.com/t/execution-environments-getting-started-guide-community-ee-images-availability/1341

`ghcr.io/ansible-community/community-ee-minimal:latest` appears to be working. Will update this PR.

File: execution-environment.yml [hunk 1]
```diff
@@ -4,16 +4,20 @@ version: 3
...
+     dest: configs
+
 additional_build_steps:
+  prepend_galaxy:
+    - ADD _build/configs/ansible.cfg ~/.ansible.cfg
   append_final:
-    - RUN curl -O https://mirror.openshift.com/pub/rhacs/assets/4.1.2/bin/linux/roxctl
-    - RUN chmod +x roxctl
-    - RUN mv roxctl /usr/local/bin/roxctl
+    - RUN cd /usr/local/bin && curl -O https://mirror.openshift.com/pub/rhacs/assets/4.1.2/bin/linux/roxctl && chmod +x /usr/local/bin/roxctl
```
Comment (bcrochet) [inline execution-environment.yml:23]:
```suggestion
  append_final: |
    RUN cd /usr/local/bin && \
            curl -O https://mirror.openshift.com/pub/rhacs/assets/4.1.2/bin/linux/roxctl && \
            chmod +x /usr/local/bin/roxctl
```

Comment (bcrochet) [reply execution-environment.yml:23]: This would be preferred, but also should probably just be in a separate PR.

**Review by bcrochet — APPROVED** [review]
/lgtm

**Pull request merged**